backend/yonder/views.py

```diff
@@ -301,11 +301,25 @@ def retrieve(self, request, author_id, follower_id):
         return Response(status=status.HTTP_404_NOT_FOUND)
 
     def destroy(self, request, author_id, follower_id):
-        author_follower = get_object_or_404(AuthorFollower, author=author_id, follower__id=follower_id)
-        author_follower.delete()
+        try:
+            author_follower = AuthorFollower.objects.get(author=author_id, follower__id=follower_id)
+            author_follower.delete()
+            return Response(status=status.HTTP_204_NO_CONTENT)
+            
+        except AuthorFollower.DoesNotExist:
 
-        return Response(status=status.HTTP_204_NO_CONTENT)
+            authorToUnfollow = request.data["actor"]
+            node = RemoteNode.objects.get(host=authorToUnfollow["host"])
+            url = node.host + "api/author/" + str(authorToUnfollow["id"]) + "/followers/" + str(follower_id) + "/"
+            response = requests.delete(url,
+                auth=requests.models.HTTPBasicAuth(node.our_user, node.our_password),
+                json=request.data, 
```

Comment:
other servers dont expect any body for DELETE, so you can remove this and the below headers.